### PR TITLE
Update auth instructions.

### PR DESCRIPTION
**Pull request by jfree**
### What does this PR do?
Updates the section describing initial account setup for the IntelliJ IDEA plugin.

### Motivation
We have a new beta release coming up soon where the plugin switches to OAuth instead of using API and App keys.

### Additional Notes
None.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/developers/ide_integrations/idea/_index.md [hunk 1]
```diff
@@ -50,14 +50,14 @@ In addition to code highlighting, you can use the Datadog plugin for Intellij ID
...
-### Add Datadog credentials
+### Log in to Datadog
 
-After installing the Datadog plugin and restarting IDEA, add your Datadog API and application keys to authenticate with Datadog:
+After installing the Datadog plugin and restarting IDEA, log in to Datadog:
 1. With a file or project open in IDEA, click the **Datadog** tool window.
-1. Click **Add your credentials...**.
-1. Enter your name, [API key and application key][5], and select a site.
+1. Click **Log in...**.
+1. In the browser window that opens, select your site and organisation then authorize access to the platform.
```
Comment (urseberry) [inline content/en/developers/ide_integrations/idea/_index.md:58]:
```suggestion
1. In the browser window that opens, select your site and organization, then authorize access to the platform.
```

File: content/en/developers/ide_integrations/idea/_index.md [hunk 2]
```diff
@@ -50,14 +50,14 @@ In addition to code highlighting, you can use the Datadog plugin for Intellij ID
...
 
-**Note**: For most users, one pair of keys is all that is required. If you're using a multi-org setup with multiple key pairs, check to ensure that the correct pair is active. To find out which key IDEA is using, click **Preferences** -> **Tools** -> **Datadog** and check which account is active.
+**Note**: For most users, one login is all that is required. If you're using a multi-org setup, check to ensure that the correct account is active. To find out which login IDEA is using, click **Preferences** -> **Tools** -> **Datadog** and check which account is active.
```
Comment (urseberry) [inline content/en/developers/ide_integrations/idea/_index.md:60]:
```suggestion
**Note**: For most users, one login is all that is required. If you're using a multi-org setup, check to ensure that the correct account is active. To find out which login IDEA is using, click **Preferences** -> **Tools** -> **Datadog**, and check which account is active.
```

File: content/en/developers/ide_integrations/idea/_index.md [hunk 3]
```diff
@@ -156,5 +156,5 @@ Let us know what you think about the plugin! Provide feedback on our [discussion
 [1]: https://github.com/DataDog/datadog-for-intellij/discussions
 [3]: /getting_started/profiler/
 [4]: https://www.datadoghq.com/
-[5]: /account_management/api-app-keys/
-[6]: https://plugins.jetbrains.com/plugin/19495-datadog
\ No newline at end of file
+[5]: /getting_started/site/#overview
```
Comment (urseberry) [inline content/en/developers/ide_integrations/idea/_index.md:159]: It looks like this link is no longer used and can be removed.

**Review by urseberry — COMMENTED** [review]
You said that the upcoming beta release causes the authorization method to switch. When does the release go out? Should we delay this documentation update until the change is live?

File: content/en/developers/ide_integrations/idea/_index.md (hunk 3 above)
Comment (jfree) [reply content/en/developers/ide_integrations/idea/_index.md]: Good spot, fixed.

**Review by urseberry — APPROVED** [review]

**Pull request merged**